aion/lib/aion/room_channel/room.ex

```diff
@@ -6,13 +6,19 @@ defmodule Aion.RoomChannel.Room do
   alias Aion.RoomChannel.{Room, UserRecord}
   require Logger
 
+  @type question_list :: (list Question.t)
+
+  @type new_question_set :: %{questions: question_list, question: Question.t, answers: list Answer.t}
+
   @type t :: %__MODULE__{users: %{String.t => UserRecord.t},
                          users_count: integer,
+                         questions: question_list,
```

Comment:
If you exported the type to `questions.ex`, you would end up having `questions: Questions.t` here and you could rename `question` to `current_question: Question.t`. I think it would look better.